std/typecons.d

```diff
@@ -5189,6 +5189,8 @@ mixin template Proxy(alias a)
                 return a.opCmp(b);
             else static if (is(typeof(b.opCmp(a))))
                 return -b.opCmp(a);
+            else static if (isFloatingPoint!ValueType || isFloatingPoint!B)
+                return a < b ? -1 : a > b ? +1 : a == b ? 0 : float.nan;
```

Comment:
Given `Typedef!float x,y;`, it's the code that's generated when you do this:
`bool a = x [<>]=? y;`
that matters, not the code that is generated when you do this:
`int b = x.opCmp(y);`

That is what I am testing in the link I gave.